File: Abogado.java
package Trabajo;

public class Abogado {
    public String nombre;

    public String genero;

    public String edad;

    public Abogado(String nom, String gen, String ed)
    {
        establecerNombre(nom);
        establecerGenero(gen);
        establecerEdad(ed);
    }

    //nombre
    public String obtenerNombre()
    {
        return nombre;
    }

    public void establecerNombre(String nom)
    {
        nombre = nom;
    }

    //genero
    public String obtenerGenero()
    {
        return genero;
    }

    public void establecerGenero(String gen)
    {
        genero=gen;
    }

    //edad
    public String obtenerEdad()
    {
        return edad;
    }
    public void establecerEdad(String ed)
    {
        edad = ed;
    }
}

File: Abogado.php
<?php

namespace App\Model;

class Abogado
{
    public $nombre = "";
    public $movil = "";
    public $peso = "";

    //nombre
    public function __construct($nom)
    {
       $this->establecerNombre();
    }

    public function obtenerNombre()
    {
        return $this->nombre;
    }

    public function establecerColor($nom)
    {
        $this->nombre = $nom;
    }

    //movil
    public function __construct($mov)
    {
       $this->establecerMovil();
    }

    public function obtenerMovil()
    {
        return $this->movil;
    }

    public function establecerMovil($mov)
    {
        $this->movil = $mov;
    }

    //peso
    public function __construct($pes)
    {
       $this->establecerPeso();
    }

    public function obtenerPeso()
    {
        return $this->peso;
    }

    public function establecerPeso($pes)
    {
        $this->peso = $pes;
    }
}